lakeview/src/main/java/ai/onehouse/config/models/common/GCSConfig.java

```diff
@@ -13,4 +13,5 @@
 public class GCSConfig {
   @Builder.Default private Optional<String> projectId = Optional.empty();
   @Builder.Default private Optional<String> gcpServiceAccountKeyPath = Optional.empty();
+  @Builder.Default private Optional<String> destinationServiceAccount = Optional.empty();
```

Comment:
Nit we can rename this to serviceAccountToImpersonate or something similar? Same for AWS as well